orchestra/scripts/add_slack_user_ids.py

```diff
@@ -0,0 +1,49 @@
+from annoying.functions import get_object_or_None
```

Comment:
i assume this script won't run b/c django isn't configured?  make it a management command?